src/services/LiquidityService.js

```diff
@@ -225,11 +225,29 @@ check should be done`,
           params: [ auctionIndex, tokenA, fundingA, tokenB, fundingB ],
           notify: true
         })
-        // Do sell in the correct auction
-        const soldToken = await this._sellTokenToCreateLiquidity({
-          tokenA, fundingA, tokenB, fundingB, auctionIndex, from
-        })
-        soldTokens.push(soldToken)
+
+        let soldTokenAB, soldTokenBA
+        // Ensure liquidity for both sides or the side that needs it
+        if (fundingA.lessThan(minimumSellVolume)) {
+          soldTokenAB = await this._sellTokenToCreateLiquidity({
+            sellToken: tokenA,
+            buyToken: tokenB,
+            funding: fundingA,
+            auctionIndex,
+            from
+          })
+          soldTokens.push(soldTokenAB)
+        }
+        if (fundingB.lessThan(minimumSellVolume)) {
+          soldTokenBA = await this._sellTokenToCreateLiquidity({
+            sellToken: tokenB,
+            buyToken: tokenA,
+            funding: fundingB,
+            auctionIndex,
+            from
+          })
+          soldTokens.push(soldTokenBA)
```

Comment:
Now we'll start to send two transactions at the same time. I hope the new mechanisms for the nonce works as expected :)